### DISCUSSION TITLE
articles/comment-via-giscus

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#150] [Comments]
# articles/comment-via-giscus

我的博客引擎用的是 Jekyll，一款 “古老” 的静态博客引擎。写下 “古老” 这个词，我不禁感叹时间飞逝，回想当时刚用 Jekyll 时，那时可是新东西。在那个年代，如果不是在 Blogspot、新浪博客这种博客托管平台上写博客而是想自建的话，wordpress 几乎是不二选择。可以说，wordpress 几乎是博客软件的代名词。而 Jekyll 的发明可以说是创造性的，用作者 Tom Preston-Werner（同时也是 GitHub 的创始人）的话说就是用 Jekyll 写博客就“像黑客一样写博客”(Blogging Like a Hacker)。从此，自建博客系统不需要关系服务器，不需要配置数据库，只需要懂 Git、熟悉 Markdown 语法就可以拥有一个完美的程序员的博客。

https://hutusi.com/articles/comment-via-giscus

**Comment by DeppWang** [comment]
来来来，灌水评论一下，哈哈

**Reply by hutusi** [reply]
欢迎灌水，哈哈😄

**Reply by pengwon** [reply]
也过来试试看怎么灌水😄

**Reply by lizilong1993** [reply]
学习啦哈哈，这个感觉确实比gittalk好用，我也去改一下我的博客评论系统2333

**Reply by Meiting-Wang** [reply]
come baby

**Comment by 17678319606** [comment]
已经添加友情链接

**Reply by hutusi** [reply]
🤝

**Comment by miao1007** [comment]
此文写的何等的好啊，已经用上

**Reply by hutusi** [reply]
哈哈，此话听来何等的熟悉啊😀

**Comment by bearfm** [comment]
我也来凑个热闹试试这个评论系统

**Reply by hutusi** [reply]
cool！域名很酷，博客也很酷。加个友链吧？😁

**Reply by bearfm** [reply]
这是什么样的运气，得到大神的青睐啊

**Reply by hutusi** [reply]
你太谦虚了😀

**Comment by qichunren** [comment]
已经订阅，加油写。

**Reply by hutusi** [reply]
谢谢😀

**Comment by ZhangJun1015** [comment]
也想试一下静态博客，就是评论这一块感觉比较麻烦。

**Reply by hutusi** [reply]
用惯了静态博客其实还蛮喜欢 Disqus 或用 github issues/discussions 来管理评论这种方式的。特别对于评论者来说，我觉得这样更好，因为可以看到自己在哪些网站上发表过什么评论，也容易跟别人产生互动。

**Reply by bearfm** [reply]
我来砸场子，你也可以试一试twikoo

**Reply by hutusi** [reply]
> 我来砸场子，你也可以试一试twikoo

看介绍像 Valine

**Comment by BakaRice** [comment]
(๑•̀ㅂ•́)و✧棒

**Reply by hutusi** [reply]
😊

**Comment by xibolun** [comment]
改天我也换一下 : )

**Reply by hutusi** [reply]
挺好用的！😀

**Comment by chenyukang** [comment]
不要给用户过多选择，哈哈~

**Reply by hutusi** [reply]
哈哈，还好只是二选一😀

**Comment by forecho** [comment]
我上个月月底把评论从 utterances 换成 giscus 了 https://github.com/forecho/hugo-blog/commit/72bd8fc1a950a11b41b8582cc18f357822c4ac0c

**Reply by hutusi** [reply]
殊途同归😀

**Comment by schwannoma** [comment]
博客做的很棒！！支持一下！我也在搭建自己的主页 XD

**Reply by hutusi** [reply]
谢谢~~搭好加友链😀

**Comment by fzdwx** [comment]
cool

**Reply by hutusi** [reply]
😊

**Comment by schwannoma** [comment]
***@***.***_21，

**Comment by iamgodot** [comment]
相比 gittalk 抄家般的授权要求，这个有安全感多了，而且 reactions 也很实用。

本来觉得 waline/twikoo 那样的评论系统对用户更友好，但作为开发者真的很难对 GitHub 说不。。

**Reply by hutusi** [reply]
GitHub 确实很方便，不过唯一不好的是在国内访问网络不太稳定，有时会连不上。之前用 gitalk 的时候很容易出现评论加载失败的情况，现在换了 giscus 后好一些了。

**Reply by iamgodot** [reply]
感觉跟 app 本身关系不大？另外就是 giscus 不需要手动设置 token 之类的所以用起来极其方便，我现在已经真香了哈哈，感谢推荐

**Reply by hutusi** [reply]
跟 app 关系不大，就是网络问题。新工具总是要比老的工具好用些~

**Reply by lly-ke** [reply]
效果不错
---
~~测试~~

**Comment by Emily-0616** [comment]
打卡

**Comment by ufeer** [comment]
我也来打个卡，大佬好强

**Reply by hutusi** [reply]
不敢当大佬😂

**Comment by YeFei572** [comment]
貌似确实比gitalk牛皮一些啊

**Reply by hutusi** [reply]
感觉好用些~

**Comment by ChestnutHeng** [comment]
好耶

**Comment by Moosphan** [comment]
这年头用jekyll的真的不多了，刚好也在换评论系统，mark一下😁

**Reply by hutusi** [reply]
嗯，Ruby程序员越来越少了~

**Comment by optimjie** [comment]
若不是新建discussion的话，是不是没有邮件提醒呢？

**Reply by optimjie** [reply]
不对，是有邮件提醒的，有点尴尬。

**Reply by hutusi** [reply]
我一般就看github 的notification

**Comment by pluswave** [comment]
搜索到这篇文章，把[自己博客](https://meixin.chat)的评论系统也换了，谢谢你！

**Reply by hutusi** [reply]
谢谢，你的博客内容不错，加油 💪

**Comment by chrisyy2003** [comment]
测试

**Comment by chrisyy2003** [comment]
这个评论之后，别人不是回复我的好像也能收到消息

**Comment by residualsun1** [comment]
你好！想请问如果没有内置模板里面没有配置giscus，可以自己手动加入吗？

**Reply by hutusi** [reply]
可以自行配置，按照giscus的官方指导操作即可。

**Reply by residualsun1** [reply]
好的，谢谢你！

**Comment by Tennisatw** [comment]
学到了

**Comment by sundegan** [comment]
最好的评论系统，满足我所有的要求

**Comment by Meiting-Wang** [comment]
来，灌灌水

**Comment by xiaoheige** [comment]
好好好

**Comment by bg51717** [comment]
当我在一个页面下进行评论的时候，之前所有评论的人都会收到邮件通知。作者有遇到这个情况吗？

**Reply by hutusi** [reply]
我自己都收不到邮件通知 😆

**Comment by Eggest** [comment]
六六六，为啥我的博客用不了gitalk啊，老是显示 Error: Request failed with status code 405 qwq
求助qwq

**Reply by hutusi** [reply]
建议到gitalk的项目发个issue求助看看~
